docs/features/techdocs/configuring-ci-cd.md

```diff
@@ -173,8 +173,7 @@ jobs:
         run: techdocs-cli generate --no-docker --verbose
 
       - name: Publish docs site
-        run:
-          techdocs-cli publish --publisher-type awsS3 --storage-name
+        run: techdocs-cli publish --publisher-type awsS3 --storage-name
           $TECHDOCS_S3_BUCKET_NAME --entity
```

Comment:
should probably be careful with multiline strings on this form, better then to explicitly do

```yaml
run: >-
  techdocs-cli
  $...
```

Reply:
switched this to having it all on a single line instead